app/Lib/BulkUploader.php

```diff
@@ -89,7 +89,7 @@ class BulkUploader
                 [
                     'id' => 'tickets',
                     'label' => 'Create Access Documents',
-                    'help' => "callsign,type[,wap date]\ntype = cred (Staff Credential), rpt (Reduced-Price Ticket), gift (Gift Ticket), vp (Vehicle Pass), wap (Work Access Pass)",
+                    'help' => "callsign,type[,wap date]\ntype = cred (Staff Credential), rpt (Reduced-Price Ticket), gift (Gift Ticket), vp (Vehicle Pass), wap (Work Access Pass)\n\nAdvanced usage: callsign,type[,wap date,source year, expiry year]\nsource year and expiry year are only supported for cred and rpt",
```

Comment:
Use "access date" instead of "wap date" since "wap" is specific terminology for WAPs. However both WAPs and Staff Credentials have access dates.